test/unit/spec/PrimaryAuth_spec.js

```diff
@@ -466,6 +481,13 @@ Expect.describe('PrimaryAuth', function() {
   });
 
   Expect.describe('elements', function() {
+    beforeEach(() => {
+      // BaseLoginRouter will render twice if language bundles are not loaded:
+      // https://github.com/okta/okta-signin-widget/blob/master/src/util/BaseLoginRouter.js#L202
+      // We are not testing i18n, so we can mock language bundles as loaded
+      Util.mockBundles();
```

Comment:
Since this is a race condition related to the beacon, let's move this into the test below so it doesn't extend to all other tests.